app/languages/en/index.js

```diff
@@ -9,6 +9,8 @@ import DownloadMap from './user/downloadMap.json';
 import ManageUser from './user/manageUser.json';
 import SavedAreas from './user/savedAreas.json';
 import Permission from './static/permission.json';
+import Review from './user/treeReview.json';
```

Comment:
I think this needs to be changed in every of these similar files in every language folder.

Reply:
Yes. it needs to be translated

Reply:
I meant there is an index.js file in every language folder which needs to be changed like this file.